modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, rank, nthread):
             f"Actor <{self._rank}>, nthread = {self._nthreads} was initialized."
         )
 
-    def _get_dmatrix(self, X_y):
+    def _get_dmatrix(self, X_y, **dmatrix_params):
```

Comment:
```suggestion
    def _get_dmatrix(self, X_y, **dmatrix_kwargs):
```

Reply:
this was rename

Reply:
This should be renamed in all such places.